lesson16/main.js

```diff
@@ -91,31 +102,33 @@ function createTab({ category, id, select }) {
   fragmentTabs.appendChild(tabTitle).appendChild(tabAnchor);
 }
 
-function createArticles({ article }) {
-  const articles = article;
-  for (const article of articles) {
+function createArticles(articles) {

```

Comment:
IMO
ここでcreateしているのは単数のarticleタイトルだと思います。
```suggestion
function createArticle(articles) {

```

or
fragmentTitlesに合わせてtitleという文言を使うのもいいと思います
```suggestion
function createArticleTitle(articles) {

```

Reply:
Please teach me(. .)
Generaly, Do we name any function singular when the function create something one by one with loop?

![image](https://github.com/groovywave/JS_Practice/assets/76876597/6eb74cb0-e1d3-48f2-a6af-9debe0f8e5c9)

Reply:
> Generaly, Do we name any function singular when the function create something one by one with loop?

I'm not sure if it's common, but I think there are many such cases

Reply:
Thank you sae-san :）
I changed the function's name.
7697dad77b090711fbc641d222d313a3402d46b0